lxqt-admin-time/CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ lxqt_translate_desktop(DESKTOP_FILES
 lxqt_app_translation_loader(lxqt-admin-time_QM_LOADER ${PROJECT_NAME})
 #************************************************
 
+find_package(Systemd)
```

Comment:
@chris2553 Where is the Systemd CMake Module ?
LXQt doesn't provide one, CMake also doesn't provide it and the systemd package itself also doesn't provide it.
`find_package()` will always fail and `lxqt-admin-time` will always be built without Systemd support.